src/Stratis.Bitcoin.Features.Miner.Tests/PosBlockAssemblerTest.cs

```diff
@@ -126,7 +127,7 @@ public void CreateNewBlock_WithScript_ReturnsBlockTemplate()
         [Fact]
         public void CreateNewBlock_WithScript_DoesNotValidateTemplateUsingRuleContext()
         {
-            var newOptions = new PosConsensusOptions() { MaxBlockWeight = 1500 };
```

Comment:
This weight wasn't actually functioning - it wasn't being applied in BlockDefinition anyway (min in constructor was 4000 I believe).

Actually using a weight of 1500 is nonsensical -> it's less than the weight of a block anyway